### ISSUE TITLE
Song from previous album showing up in the next album clicked as well

### ISSUE THREAD

**Issue by sametaor** [josephbeattie/Fluent-Media-Player-Dev#103]
Whenever I try to play music from any album, it shows the appropriate song(s), but when I change over to another album, it shows the song(s) from that previous album as well, as seen in this recording:-

https://user-images.githubusercontent.com/71749831/147965505-dc62b477-8848-4f36-a62e-12a355c6d415.mp4

Just to clarify, the app does not crash after I press play on that song, it waits for sometime, and then gives a blank now playing bar instead to start over.

## Steps to Reproduce:-
__Step 1:__ Play a song from an album.
__Step 2:__ Go to another album.
__Step 3:__ The song from the former album will appear instead of the appropriate one(s).

**Comment by isaiahcabugos** [comment]
Unable to reproduce this bug using the most recent github code. Has this issue been fixed?

**Comment by josephbeattie** [comment]
Yes, appears to be. We have worked on the indexer, so it could be down to that, and some changes have been made to ‘albums by this artist’.
